apps/arweave/src/ar_sync_record.erl

```diff
@@ -131,8 +132,10 @@ delete(End, Start, ID, StoreID) ->
 	end.
 
 delete_async(Event, End, Start, ID, StoreID) ->
+	delete_async(Event, End, Start, ID, StoreID, none).
+delete_async(Event, End, Start, ID, StoreID, ReportErrorsTo) ->
 	GenServerID = name(StoreID),
-	gen_server:cast(GenServerID, {delete_async, Event, End, Start, ID}).
+	gen_server:cast(GenServerID, {delete_async, Event, End, Start, ID, ReportErrorsTo}).
```

Comment:
Did you add ReportErrorsTo for debugging purposes?

Reply:
I can probably remove this. In an earlier iteration I'd used delete_async to try to optimize the repacking. And then added REportErrorsTo as a way to be notified if there was a problem so that ar_repack could then invalidated that chunk (rather than leave it in an intermediate state)

However I backed that change out cause it didn't make a difference in performance but dded a lot of complexity. I just did a quick scan and it turns out we don't use delete_async anywhere in the codebase. So maybe I'll just remove the function altogether?